automation/build.sh

```diff
@@ -68,7 +68,7 @@ cd "$COLLECTION_DIR"
 
 ansible-test sanity
 antsibull-changelog lint -v
-ansible-lint roles/* --exclude roles/hosted_engine_setup --exclude roles/disaster_recovery --exclude roles/remove_stale_lun -x experimental
+ansible-lint roles/* --exclude --exclude roles/disaster_recovery --exclude roles/remove_stale_lun -x experimental
```

Comment:
Shouldn't also the empty `--exclude` option be removed?

Reply:
Yes...